.github/workflows/commit_actions.yaml

```
@@ -3,10 +3,25 @@ run-name: ${{ github.actor }} is building and releasing (to github) the mac,wind
     
 jobs:
+  download-assets:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create models directory
+        run: mkdir -p client/app/models
+      - name: Download Model
+        run: |
+          curl -L -o client/app/models/google-gemma-3-4b-it https://gcs-file-downloader-10805446439.us-central1.run.app/download/google-gemma-3-4b-it
+      - name: Upload models artifact
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.8)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)